Packages/com.unity.inputsystem/Documentation~/Layouts.md

```diff
@@ -40,7 +40,7 @@ In its most basic form, a layout can be expressed by a C# class derived from:
 
 ```CSharp
 // The InputControlLayout attribute is not strictly necessary here.
-// However, it can be used to set additional properties (such as
+// However, it can set additional properties (such as
```

Comment:
The attribute itself doesn't set anything, the user does - can we change this to "you can use it to set..."?